CRM/Core/DAO/Website.php

```diff
@@ -124,19 +124,25 @@ public static function &fields() {
           'entity' => 'Website',
           'bao' => 'CRM_Core_BAO_Website',
           'localizable' => 0,
+          'html' => [
+            'type' => 'Number',
+          ],
           'add' => '3.2',
         ],
         'contact_id' => [
           'name' => 'contact_id',
           'type' => CRM_Utils_Type::T_INT,
-          'title' => ts('Contact'),
+          'title' => ts('Contact ID'),
           'description' => ts('FK to Contact ID'),
           'where' => 'civicrm_website.contact_id',
           'table_name' => 'civicrm_website',
           'entity' => 'Website',
           'bao' => 'CRM_Core_BAO_Website',
           'localizable' => 0,
           'FKClassName' => 'CRM_Contact_DAO_Contact',
+          'html' => [
+            'label' => ts("Contact"),
```

Comment:
Apologies in advise for commenting on a merged PR and spamming everyone.

But I was trying to track down a field label inconsistency in the Data Processor, https://lab.civicrm.org/extensions/dataprocessor/-/issues/90

Why is this field label "Contact" and not "Contact ID" ?

Reply:
The difference between "title" and "label" is that the title describes what the field contains (Contact ID) and the label is how a form label would appear (a Select field labeled "Contact").

Reply:
cool thanks @colemanw